back/engines/commercial/multi_tenancy/config/intercom-event-whitelist.txt

```diff
@@ -23,16 +23,13 @@ Idea changed_body
 Idea changed_status
 Idea changed_title
 Idea deleted
-Idea first published by user
-Idea first_published_by_user
 Idea published
 IdeaAssignment::Notifications::IdeaAssignedToYou created
 Initiative changed
 Initiative changed_body
 Initiative changed_status
 Initiative changed_title
 Initiative deleted
-Initiative first_published_by_user
```

Comment:
Removing similar and code related to first Initiative published (which isn't in use)